bugbug/model.py

```diff
@@ -204,18 +241,34 @@ def train(self, importance_cutoff=0.15):
 
             matplotlib.pyplot.savefig("feature_importance.png", bbox_inches="tight")
 
-            # TODO: Actually implement feature importance visualization for multiclass problems.
+            avg_shap_values = 0
             if isinstance(shap_values, list):
-                shap_values = np.sum(np.abs(shap_values), axis=0)
+                avg_shap_values = np.sum(np.abs(shap_values), axis=0)
+            else:
+                avg_shap_values = shap_values
+                shap_values = [shap_values]
 
             important_features = self.get_important_features(
-                importance_cutoff, shap_values
+                importance_cutoff, avg_shap_values, shap_values, class_names
             )
 
-            print(f"\nTop {len(important_features)} Features:")
-            for i, [importance, index, is_positive] in enumerate(important_features):
+            print("Top {} features:".format(len(important_features["average"])))
+            top_feature_names = [
+                feature_names[index] for index in important_features["average"]
+            ]
+
+            # allow maximum of 8 columns in a row to fit the page better
```

Comment:
We should share as much as possible this code between model.py and run.py. You can add a new function "print_feature_importances".